app/steps/ui/will/newleft/index.js

```diff
@@ -0,0 +1,21 @@
+const ValidationStep = require('app/core/steps/ValidationStep'),
+    json = require('app/resources/en/translation/will/left.json');
```

Comment:
Please put each variable declaration on a new line, see https://github.com/hmcts/probate-tech-docs/blob/master/javascript-coding-style.md#variable-declarations for an example